scripts/management/commands/secret_santa.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+import random
+
+from django.core.management.base import BaseCommand
+from django.core.mail import EmailMessage
+from django.conf import settings
+from django.template.loader import render_to_string
+
+from apps.events.models import AttendanceEvent
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('event_id')
+
+
+    def handle(self, *args, **options):
+        event_id = options['event_id']
+        event = AttendanceEvent.objects.get(event=event_id)
+        users = event.attendees_qs
+        random_users = Command.generate_random_list(users)
+        to_from_dict = {}
+        for count, user in enumerate(users):
+            to_from_dict['Par %s' % (count+1)] = {'To': user.user.get_full_name(), 'From': users[random_users[count]].user.get_full_name()}
+            
+            Command.send_mail(user, users[random_users[count]])
+
+        Command.mail_to_committee(to_from_dict)
+
+
+    # Function for generating a list with random numbers corresponding to
+    # the indexes of the list of users attending the event.
+    def generate_random_list(users):
+        to_from = random.sample(range(0, len(users)), len(users))
+        while(not Command.check_random_list(to_from)):
+            to_from = random.sample(range(0, len(users)), len(users))
+        return to_from
+
+
+    # Function for checking that the random list of indexes is valid.
+    # Checks the index of the original list of users against the randomly
+    # generated number. If index and random number is the same, then someone
+    # got themselves as secret Santa.
+    def check_random_list(random):
```

Comment:
I think the for loop can be simplified to 

 ```python
for count, user in enumerate(random):
    if count == user:
        return False
```

And maybe use something like `user_index, user2_index` instead of `count, user` to better reflect what the variables are.